### PR TITLE
Add is valid move

### LINKED ISSUE #42: Add is_valid_move(column) to Gameboard
- It should return `True`/`False` if a player can play in that column.
- Must be fast (just checking for a `None` in the top row should do it).

### PR DESCRIPTION
**Pull request by apojomovsky**
Fixes #42

### CODE REVIEW

File: game/gameboard.py [hunk 1]
```diff
@@ -71,6 +71,11 @@ def undo_move(self):
         move = self._moves.pop(-1)
         self._matrix[move[0]][move[1]] = None
 
+    def is_valid_move(self, column_index):
+        if self._matrix[0][column_index - 1]:
```
Comment (basicNew) [inline game/gameboard.py]: @apojomovsky we have been here before: `return self._matrix[0][column_index - 1] is None`

File: tests/test_gameboard.py [hunk 2]
```diff
@@ -526,3 +526,27 @@ def _test_winner_in_diagonals(self, winner_method):
         B R R R . R B
         """
         self.assertEquals(getattr(board, winner_method)(), 'blue')
+
+    def test_is_valid_move_on_valid_column(self):
```
Comment (basicNew) [inline tests/test_gameboard.py]: Couple of things: 

- The tests and the ASCII charts are swapped (sample in 1 belongs to test 2 and viceversa).
- You are missing the test with the empty column.

**Review by basicNew — COMMENTED** [review]
Couple of comments
